src/utils/executor.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as cp from 'child_process';
+import * as vscode from 'vscode';
+import { localize } from '../localize';
+
+const terminal: vscode.Terminal = vscode.window.createTerminal('Azure Functions');
+
+export function runCommandInTerminal(command: string, addNewLine: boolean = true): void {
+    terminal.show();
```

Comment:
We don't like using terminal because then you have no idea when the command finishes